tests/fixtures/fixtures_auth.py

```diff
@@ -24,4 +25,5 @@ def auth_mock(requests_mock: req_mock.Mocker) -> up42_auth.Auth:
         url=url_get_credits_balance,
         json=constants.JSON_BALANCE,
     )
-    return up42_auth.Auth(username=constants.USER_EMAIL, password=constants.PASSWORD)
```

Comment:
Why is this change needed?

Reply:
Because returning only Auth is enough for tests that uses only the `auth.Auth` class, but not for the ones that also use workspace. e.g. test_order_from_catalog that uses the workspace mock.

![Screenshot 2024-05-22 at 13 59 34](https://github.com/up42/up42-py/assets/3291933/3dc56fb3-e045-45ba-b07b-383aba459da5)